static/app/components/alertLink.stories.tsx

```diff
@@ -0,0 +1,154 @@
+        <AlertLink priority="warning">
+          Warning alert. Something is about to go wrong, probably.
+        </AlertLink>
+      </Fragment>
+    );
+  });
+
+  story('icon', () => {
+    return (
+      <Fragment>
+        <p>
+          The <JSXProperty name="icon" value /> prop can be specified if you want to add
+          an icon to the alert. Just directly pass in the icon, like{' '}
+          <JSXNode name="IconSentry" /> for example, to the prop.
+        </p>
+        <AlertLink priority="warning" icon={<IconSentry />}>
+          Read the docs.
+        </AlertLink>
+      </Fragment>
+    );
+  });
+
+  story('withoutMarginBottom', () => {
+    return (
+      <Fragment>
+        <p>
+          The <JSXProperty name="withoutMarginBottom" value /> prop is a boolean that's{' '}
+          <code>false</code> by default.
+        </p>
+        <AlertLink withoutMarginBottom>This one has no bottom margin.</AlertLink>
+        <AlertLink>This one has bottom margin by default.</AlertLink>
+        <AlertLink withoutMarginBottom>This one has no margin bottom.</AlertLink>
+      </Fragment>
+    );
+  });
+
+  story('size', () => {
+    return (
+      <Fragment>
+        <p>
+          The <JSXProperty name="size" value /> prop can be either <code>"small"</code> or{' '}
+          <code>"normal"</code>.{' '}
+        </p>
+        <AlertLink priority="info" size="normal">
+          Normal
+        </AlertLink>
+        <AlertLink priority="info" size="small">
+          Small
+        </AlertLink>
+      </Fragment>
+    );
+  });
+
+  story('to vs href vs onClick', () => {
+    const [count, setCount] = useState(0);
+    return (
+      <Fragment>
+        <p>
+          There are three ways to specify what should happen when the{' '}
+          <JSXNode name="AlertLink" /> is clicked.{' '}
+        </p>
+        <p>
+          The <JSXProperty name="to" value /> prop should be used for internal links.
+          Note: links specified with this prop will never open in a new tab, even if you
+          set
+          <JSXProperty name="openInNewTab" value="true" /> .
+        </p>
+        <AlertLink
+          priority="info"
+          size="normal"
+          to="stories/?name=app/components/badge.stories.tsx"
+        >
+          View the badge story page
+        </AlertLink>
+        <p>
+          The <JSXProperty name="href" value /> prop should be used for external links.
+        </p>
+        <AlertLink
+          priority="info"
+          size="normal"
+          href="https://sentry.io/for/session-replay/"
+          openInNewTab
+        >
+          Learn more about Session Replay
+        </AlertLink>
+        <p>
+          Lastly, you can get creative with the <JSXProperty name="onClick" value /> prop.
+        </p>
+        <AlertLink
+          priority="info"
+          size="normal"
+          href="https://sentry.io/for/session-replay/"
+          openInNewTab
+          onClick={() => setCount(count + 1)}
```

Comment:
Is it possible to have an `onClick` without the `href`?

If yes, then i'd just do `onClick` in the example, so we can see the counter incrementing.

```suggestion
          onClick={() => setCount(count + 1)}
```

Reply:
yup just updated